src/store/reducres.js

```diff
@@ -0,0 +1,17 @@
+const initialState = {
+  tasks: [],
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "ADD_TASK":
+      return {
+        ...state,
+        tasks: [...state.tasks, action.payload],
```

Comment:
In the design, a newly created task is always at the beginning.

```suggestion
        tasks: [action.payload, ...state.tasks],
```